maps/tether/tether-05-station1.dmm

```diff
@@ -8051,9 +8051,8 @@
 	opacity = 0
 	},
 /obj/machinery/door/airlock/engineering{
-	name = "Engineering Hard Storage";
-	req_access = list(11);
-	req_one_access = newlist()
+	name = "Gravity Generator";
+	req_access = list(11)
```

Comment:
Also may want to add an air alarm for the little hallway between the Grav Genny and maints.

Reply:
But there's no pumps or anything for it to control

Reply:
Then add 'em? :3

Reply:
it's a tiny 2x2 room with a single light bulb, it doesn't even have an area, it's not worth it